src/renderer/api/kube-object.ts

```diff
@@ -40,6 +40,26 @@ export interface IKubeObjectMetadata {
   }[];
 }
 
+export interface IKubeStatus {
```

Comment:
Probably having these props definitions on the class should be enough (class could be used as a type as well)